pkg/cfn/manager/api.go

```diff
@@ -88,6 +88,9 @@ func (c *StackCollection) DoCreateStackRequest(i *Stack, templateData TemplateDa
 	input := &cloudformation.CreateStackInput{
 		StackName: i.StackName,
 	}
+	if c.provider.CloudFormationOnFailureDoNothing() {
+		input.OnFailure = aws.String(cloudformation.OnFailureDoNothing)
+	}
```

Comment:
Alternatively:

```suggestion
  input.DisableRollback = aws.Bool(c.provider.CloudFormationOnFailureDoNothing())
```

Reply:
true, switched it over and just renamed to `--cfn-disable-rollback`